tensorboard/components/tf_data_selector/tf-data-selector.html

```diff
@@ -32,47 +33,21 @@
 -->
 <dom-module id="tf-data-selector">
   <template>
-    <tf-data-select-row></tf-data-select-row>
-
-    <div id="exp-add">
-      <template is="dom-if" if="[[_showExperimentAdder]]">
-        <div class="exp-selector">
-          <h4>Select data for comparison</h4>
-          <tf-filterable-checkbox-list
-            all-toggle-disabled
-            coloring="[[_experimentColoring]]"
-            items="[[_experiments]]"
-            label="Experiment"
-            max-items-to-enable-by-default="1"
-            selected-items="{{_selectedExperiments}}"
-          ></tf-filterable-checkbox-list>
-        </div>
-      </template>
-      <div class="buttons">
-        <template is="dom-if" if="[[!_showExperimentAdder]]">
-          <paper-button class="add-comparison" on-tap="_toggleExperimentAdder">
-            <iron-icon class="add" icon="add-circle-outline"></iron-icon>
-            &nbsp;Add comparison
-          </paper-buutton>
-        </template>
-        <template is="dom-if" if="[[_showExperimentAdder]]">
-          <span>
-            <paper-button on-tap="_toggleExperimentAdder">
-              Cancel
-            </paper-buutton>
-          </span>
-          <span>
-            <paper-button
-              id="add-button"
-              on-tap="_addExperiments"
-              disabled$="[[!_selectedExperiments.length]]"
-            >
-              [[_getAddLabel(_selectedExperiments.*)]]
-            </paper-button>
-          </span>
-        </template>
-      </div>
-    </div>
+    <template is="dom-if" if="[[!_comparingExps.length]]">
+      <tf-data-select-row no-experiment></tf-data-select-row>
```

Comment:
If the tf-data-select-row comments say that persistenceNumber is a required property, probably best to specify 0 here explicitly.  Also makes it clearer that it's intended to share that persisted state with the first dom-repeat child below.